packages/playground/src/screens/connection.tsx

```diff
@@ -43,6 +44,12 @@ function ActiveConnectionScreen({ connection }: { connection: Connection }) {
     return null
   }
 
+  useEffect(() => {
```

Comment:
Primarily during development, path occasionally becomes undefined. This is probably an issue with fast refresh or something. Anyway, this makes sure that whenever that happens, we just go back to the Home Screen.

Reply:
ah interesting! I will keep an eye out for unexpected navigation back